mlir-tensorrt/executor/lib/Runtime/API/API.cpp

```diff
@@ -367,13 +367,14 @@ RuntimeSession::RuntimeSession(RuntimeSessionOptions options,
 //===----------------------------------------------------------------------===//
 
 AllocTracker::~AllocTracker() {
+  MTRT_DBGF("Destroying alloc tracker %p", static_cast<void *>(this));
   MTRT_DBGF("checking %u allocations", map.size());
   llvm::SmallVector<PointerInfo> ptrsToFree;
   ptrsToFree.reserve(map.size());
   for (const auto &[ptrVal, metadata] : map) {
     if (metadata->info.isInternallyManaged() &&
         metadata->externalReferenceCount.load() == 0) {
-      MTRT_DBGF("still live: 0x%lx type %d size %lu", ptrVal,
+      MTRT_DBGF("still live: 0x%lx type %d size %lx", ptrVal,
```

Comment:
Why do you want to print the size here in hex? Usually we print sizes as `%lu`

Reply:
That was an oversight. Fixed it.